tests/integration/targets/iso_customize/tasks/prepare.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Make sure the testing sub-directory does not exist
+  file:
+    path: '{{ test_dir }}'
+    state: absent
+
+- name: Create our testing sub-directory
+  file:
```

Comment:
For consistency, please use the fully qualified name everywhere (as in line 17 below).

Reply:
Fix it. Thanks